<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMSPage</class>
 <widget class="QMainWindow" name="SMSPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>短信界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电话号码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>短信内容</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="smsContent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendNum">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="smsLogLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>短信记录</string>
    </property>
   </widget>
   <widget class="QTableView" name="smsTableView">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>411</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
